tests/integration_tests/embeddings/test_huggingface_hub.py

```diff
@@ -0,0 +1,19 @@
+"""Test huggingfacehub embeddings."""
+from langchain.embeddings import HuggingFaceHubEmbeddings
+
+
+def test_huggingfacehub_embedding_documents() -> None:
+    """Test huggingfacehub embeddings."""
+    documents = ["foo bar"]
+    embedding = HuggingFaceHubEmbeddings(task="feature-extraction")
```

Comment:
@hwchase17 dunno how we feel about default model being one where the default task isn't valid (it's "sentence-similarity"). Can find a different one if that's preferred, was just matching the existing HuggingFace embeddings class

Reply:
also seems like decent amount of feature-extraction models have 2D outputs — should we automatically flatten? or make the `embed_` signatures more permissive?

Reply:
good questions. i thik the reason for the weirdness is the HF embeddings use the sentence transformers package, but this can in theory use any

I think we'll also potentially want some different handling for whether the model is sentence transformer or not? im not super sure but how about this:

1. default the task to feature-extraction (since its the only supported one, but we still let people change it if they really want to)

2. check that repo_id starts with sentence_transformer - a bit restrictive but we can worry about extending it later? start simple and tight